client/src/test/java/com/tigrisdata/db/client/model/UpdateFieldsTest.java

```diff
@@ -0,0 +1,29 @@
+package com.tigrisdata.db.client.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.charset.StandardCharsets;
+
+public class UpdateFieldsTest {
+
+  @Test
+  public void setFields() throws JsonProcessingException {
```

Comment:
nit, can you add test when fields are not set what is the json looks like?

Reply:
added test, if no set fields are set and user is trying to build, they will face an error.